### ISSUE TITLE
Stuck on "Right before trigger" on iPad Mini 2 (A1490) 

### ISSUE THREAD

**Issue by ipadmini2** [checkra1n/BugTracker#1943]
I am stuck at "Right before trigger" with all variations, I could think of:
- iPad Mini 2 (A1490)
- original Apple cable / OEM cable
- iOS 12.4.9 and 12.5 (no higher version on this device supported)
- flashed each iOS at least three times
- three different computers 
- all available USB ports (2.0 and 3.0)
- Unbuntu 20.04 USB boot on three different sticks
- checkrain 0.98 up to 0.12.2
- Checkin / uncheckin all options
EVERYTHING ends up with checkrain hanging forever (gave up after 24 hours the longest) with "Right before trigger"

ANY ideas, how to get the device jailbroken?

**Comment by ipadmini2** [comment]
> 
> 
> I am stuck at "Right before trigger" with all variations, I could think of:
> 
>     * iPad Mini 2 (A1490)
> 
>     * original Apple cable / OEM cable
> 
>     * iOS 12.4.9 and 12.5 (no higher version on this device supported)
> 
>     * flashed each iOS at least three times
> 
>     * three different computers
> 
>     * all available USB ports (2.0 and 3.0)
> 
>     * Unbuntu 20.04 USB boot on three different sticks
> 
>     * checkrain 0.98 up to 0.12.2
> 
>     * Checkin / uncheckin all options
>       EVERYTHING ends up with checkrain hanging forever (gave up after 24 hours the longest) with "Right before trigger"
> 
> 
> ANY ideas, how to get the device jailbroken?

Finally, I found a video somewhere on youtube, giving TWO very important hints:
a) obviously, it is required to use bootra1n with checkra1n - both on 0.9.8.2
b) you need to quick unplug an replug the ipad once it seems to be stuck at "right before trigger"
It tooked threee times rerunning the procedure to finally get the ipad through the jailbreaking process.

**Comment by ForgottenGods** [comment]
I have been trying with an iPad mini 2, and I have found another partial workaround. I was able to replicate it as well but in my test I did it 20 times and was only able to get it to work 3 times.

What I did was when it got at "right before trigger " I rapidly unplugged and replugged my iPad it toke about 3 times then I would see the "booting" line ,however most of time what happened was it just booted back to the setup menu (my device was reset to act as an icloud locked device)
But 3 out of the 20 times I tried it went through the whole process.
This is on a windows pc using bootra1n as I don't own a mac, also I'm seemed to have better success rate using the 32bit version of bootra1n ,even though my pc is 64bit

I hope this helps somebody as it seems to be a little more consistent. Have a good day
